data-es2016plus.js

```diff
@@ -6993,6 +6993,118 @@ exports.tests = [
       rhino1_7_15: true
     }
   },
+  {
+    name: 'Change Array by copy',
+    category: '2023 features',
+    significance: 'small',
+    spec: 'https://github.com/tc39/proposal-change-array-by-copy',
+    subtests: [
+      {
+        name: "Array.prototype.toReversed",
+        exec: function () {/*
+          var arr = [1, 2, 3];
+          return arr.toReversed()[0] === 3 && arr[0] === 1;
+        */},
+        res: {
+          babel7corejs3: babel.corejs,
+          typescript3_2corejs3: typescript.corejs,
+          ie11: false,
+          chrome109: false,
+          chrome110: true,
+          edge109: false,
```

Comment:
Recent Edge results are mirrored from chrome, no need to add them here.

Reply:
(unless they're manually confirmed, in which case it's preferred to add them here)

Reply:
In `environments.json` the edge version have theirs corresponding chrome version, would you like that I remove this edge entries like opera ?

Reply:
If you haven't manually verified a result yourself, it shouldn't be present in the file.

Reply:
Ok, I understand. For most of the browser I only check on these sites : https://caniuse.com/?search=tosorted and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted .
You always validate  by running code on real browser ?

Reply:
Yes, otherwise there's no point in having the table, everyone could just use caniuse :-)